### ISSUE TITLE
Calendar event identification 

### ISSUE THREAD

**Issue by Thezap** [noeRls/heriot-watt-calendar-exporter#5]
We should use [extended propriety](https://developers.google.com/calendar/extended-properties) of event to store the fact that this is a generated event by this application.

The description can be edited by the user and he doesn't know the special string is important.

**Comment by noeRls** [comment]
Yep, we also need to keep in mind that we already have users that use this application so we will need to be backward compatible and keep this string check for old user.
